inc/ruleimportcomputer.class.php

```diff
@@ -304,49 +304,55 @@ function findWithGlobalCriteria($input) {
          $where_entity = $input['entities_id'];
       }
 
+      $it_criteria = [
+         'SELECT' => 'glpi_computers.id',
+         'FROM'   => '', //to fill
```

Comment:
i think the from part is not filled

Reply:
Yes, it is only populated from plugin hook as far as I remember

Reply:
oh you're right.
As far as i know, only ocsinventoryng plugin uses the hook : https://github.com/pluginsGLPI/ocsinventoryng/blob/master/hook.php#L1628.

Maybe @tsmr or anyone of infotel team can precise the usage and we can do a change for supporting iterator in the hook

Reply:
As far as I remember, I've talked about this change to @tsmr already (hre or in mattermost channel), but it was previously targetting master ;)